dotCMS/src/main/java/com/dotmarketing/quartz/QuartzUtils.java

```diff
@@ -501,12 +506,11 @@ else if (sched.getTrigger(triggerName, triggerGroup) != null)
 	 * @throws SchedulerException 
 	 * 
 	 */
-	public static boolean removeJob(String jobName, String jobGroup) throws SchedulerException {
-		Scheduler sched = DotSchedulerFactory.getInstance().getSequentialScheduler();
-		boolean result1 = removeJob(jobName, jobGroup, sched);
-		sched = DotSchedulerFactory.getInstance().getScheduler();
-		boolean result2 = removeJob(jobName, jobGroup, sched);
-		return result1 | result2;
+	public static boolean removeJob(final String jobName, final String jobGroup) throws SchedulerException {
+		final Scheduler scheduler = getScheduler();
+		boolean result1 = removeJob(jobName, jobGroup, scheduler);
+
+		return result1 ;

```

Comment:
Codacy found an issue: [Consider simply returning the value vs storing it in local variable 'result1'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6213816)